$OpenBSD: patch-sysdeps_openbsd_netload_c,v 1.3 2013/11/26 21:56:52 jca Exp $

1. - we need if_var.h but ./configure fails to define HAVE_NET_IF_VAR_H
   - some declarations in netiner/in_var.h are now kernel-private

2.
From 250666219659d721377613a916d8f166c2df93ee Mon Sep 17 00:00:00 2001
From: Antoine Jacoutot <ajacoutot@gnome.org>
Date: Wed, 16 Oct 2013 15:09:16 +0000
Subject: OpenBSD: add missing header

--- sysdeps/openbsd/netload.c.orig	Sat Jul 27 14:40:23 2013
+++ sysdeps/openbsd/netload.c	Tue Nov 26 00:46:30 2013
@@ -36,12 +36,13 @@
 
 #include <sys/ioctl.h>
 
-#ifdef HAVE_NET_IF_VAR_H
 #include <net/if_var.h>
-#endif
 
 #include <netinet/in.h>
+#define _KERNEL
 #include <netinet/in_var.h>
+#undef _KERNEL
+#include <netinet6/in6_var.h>
 
 static const unsigned long _glibtop_sysdeps_netload =
 (1L << GLIBTOP_NETLOAD_IF_FLAGS) +
